test/parse-outputs/input/empty/omicsintegrator2-empty-raw-pathway-1.txt

```diff
@@ -0,0 +1 @@
+some one line error (TODO: can we reproduce the exact error? I can't.)
```

Comment:
Is the empty file just a file with a single empty line?

Reply:
Perhaps!